prometheus/summary.go

```diff
@@ -54,6 +54,9 @@ type Summary interface {
 }
 
 // DefObjectives are the default Summary quantile values.
+//
+// Deprecated: DefObjectives will be removed in v0.10. The default will be a
```

Comment:
Handy to provide these values in v0.10, even if they aren't used internally to the package?